### ISSUE TITLE
Move "Artwork" helptext for Artwork Replacement prop template to a tooltip

### ISSUE THREAD

**Issue by isaaczaak** [ourzora/nouns-builder#243]
**Is your feature request related to a problem? Please describe.**
The Artwork Replacement prop template is too wordy now. 

**Describe ideal solution**
Move the help text for "Artwork" into a tooltip.

**Comment by b3nedictvs** [comment]
The recent questions and uncertainties around artwork upload make us disagree with this ticket. We therefore close it as won't fix.
